packages/currency/src/erc20/networks/goerli.ts

```diff
@@ -0,0 +1,17 @@
+import { TokenMap } from './types';
+
+// List of the supported goerli ERC20 tokens
+export const supportedGoerliERC20: TokenMap = {
+  // Request Test token, used for testing on goerli.
+  '0x7af963cF6D228E564e2A0aA0DdBF06210B38615D': {
+    decimals: 18,
+    name: 'Goerli Test Token',
+    symbol: 'TST',
```

Comment:
I don't see that token used anywhere in the test, when do we need it?

Reply:
We don't need it I think, it was because I added two tokens FAU and that one at the beginning